### PR TITLE
refactor linting parts of #136

### PR DESCRIPTION
**Pull request by phlprcks**
see comments in #136 for details

### CODE REVIEW

**Comment by sinnaj-r** [comment]
@phlprcks is this one ready to be reviewed ? You have no reviewers assigned 😃

**Comment by phlprcks** [comment]
> 
> 
> @phlprcks is this one ready to be reviewed ? You have no reviewers assigned 😃

Oh, good call. I only mentioned it in #136 and then never requested the PR review 🙈 Just to be sure I merged dev into this branch again as there was some busy merging going on.

File: .github/workflows/smalltalk-ci.yml [hunk 1]
```diff
@@ -23,4 +23,4 @@ jobs:
       shell: bash
```
Comment (sinnaj-r) [inline .github/workflows/smalltalk-ci.yml:23]: You are reversing a commit here

File: README.md [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Master: [![Coverage Master](https://coveralls.io/repos/github/hpi-swa-teaching/Algernon-Launcher/badge.svg?branch=master)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master) Development: [![Coverage Development](https://coveralls.io/repos/github/hpi-swa-teaching/Algernon-Launcher/badge.svg?branch=development)](https://coveralls.io/github/hpi-swa-teaching/Algernon-Launcher?branch=master) 
 
 <p align="center">
-  <img src="https://user-images.githubusercontent.com/44369294/89326002-e93a5800-d689-11ea-89f0-d685e7a4f1e9.png" width="465" alt="Algernon"/>
```
Comment (sinnaj-r) [inline README.md:6]: You are reversing a commit here

File: packages/Algernon.package/ALGCore.class/instance/wasChanged..st [hunk 3]
```diff
@@ -1,3 +0,0 @@
-updating
-wasChanged: aBool
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGCore.class/instance/wasChanged..st:2]: We need this method for backwards compatibility

File: packages/Algernon.package/ALGCore.class/instance/systemChanged..st [hunk 4]
```diff
@@ -1,3 +0,0 @@
-updating
-systemChanged: evt
```
Comment (sinnaj-r) [inline packages/Algernon.package/ALGCore.class/instance/systemChanged..st:2]: We need this method for backwards compatibility

**Review by sinnaj-r — CHANGES_REQUESTED** [review]
This generally looks good 👍 
There only seems to be a problem with git reversing some commits here 🤔

**Comment by phlprcks** [comment]
as there are difficulties with the readme and yaml files even after rebasing in squeak git browser. I felt saver to implement it in a new branch that was branched from the current status in development... see #162

**Pull request closed without merging**